src/core/services/gesture/gesture.js

```diff
@@ -243,6 +244,18 @@ function MdGesture($$MdGestureHandler, $$rAF, $timeout) {
         horizontal: true,
         cancelMultiplier: 1.5
       },
+      onSetup: function(element, opts) {
+        if (touchActionProperty) {
+          // We check for horizontal to be false, because otherwise we would overwrite the default opts.
+          this.oldTouchAction = element[0].style[touchActionProperty];
+          element[0].style[touchActionProperty] = opts.horizontal === false ? 'pan-y' : 'pan-x';
```

Comment:
You can replace this one with `element.css(touchActionProperty);`, although you should keep it mind that it'll be undefined if the element doesn't have inline styles.